zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/tenant/DbTenantState.java

```diff
@@ -51,4 +53,28 @@ public DbTenantState(
         zeebeDb.createColumnFamily(
             ZbColumnFamilies.TENANT_BY_ID, transactionContext, tenantId, fkTenantKey);
   }
+
+  @Override
+  public void createTenant(final TenantRecord tenantRecord) {
+    tenantKey.wrapLong(tenantRecord.getTenantKey());
+    tenantId.wrapString(tenantRecord.getTenantId());
+    persistedTenant.setTenant(tenantRecord);
```

Comment:
💭 Just a heads-up but we're going to have to modify the `PersistedTenant` soon. Instead of storing the whole `TenantRecord` we should only store the fields that are relevant. So the `PersistedTenant` will get a few properties (tenantKey, tenantId, name). No need to change it as part of this PR, but it's good to know that it's coming.